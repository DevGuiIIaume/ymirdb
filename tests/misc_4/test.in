set a 1 2 3
set b a 1 a 1 a
sum b
len b
min b
max b
bye
