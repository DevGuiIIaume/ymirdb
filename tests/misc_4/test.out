> ok

> ok

> 20

> 11

> 1

> 3

> bye
